src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -225,23 +225,29 @@ def build_integration(self, state):
             identity['access_token'],
             state['installation_id'])
 
+        domain_name = installation['account']['html_url'].replace('https://', '').split("/")[0]
```

Comment:
We should probably just use [`urlparse`](https://docs.python.org/2/library/urlparse.html) for this.

Also in the metadata we probably don't want to trim the organization / user name, since the domain name is mostly for visual purposes in the metadata.